cypress.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
You can add a property `"baseUrl": "http://localhost:3000` here that automatically gets prepended to the url in `cy.visit()` calls. This way you don't have to refactor all your tests if the address of your test server changes.